src/snowflake/cli/api/commands/flags.py

```diff
@@ -378,6 +378,16 @@ def like_option(help_example: str):
     )
 
 
+def pattern_option(help_example: str) -> typer.Option:
+    return typer.Option(
+        ".*",
```

Comment:
Let's make it `None` to avoid any issues. If pattern is not provided don't add it to `ls` query